### PR TITLE
User redux

### PR DESCRIPTION
**Pull request by dfeeney87**
### Assignee Tasks

- [ ] added unit tests (or none needed)
- [ ] written relevant docs (or none needed)
- [ ] referenced any relevant issues (or none exist)

### Guidelines

Please add a description of this Pull Request's motivation, scope, outstanding issues or potential alternatives, reasoning behind the current solution, and any other relevant information for posterity.

---

*Your PR Notes Here*

### CODE REVIEW

File: client/components/user/user-item.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+
+const UserList = (props) => {
+  let { email, firstName, googleId, isAdmin, lastName } = props.user;
+
+  if (isAdmin) {
+    isAdmin = 'Admin User'
```
Comment (davidfudacz) [inline client/components/user/user-item.js]: call this something other than isAdmin

File: client/components/user/user-item.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+  } else {
+    isAdmin = 'Normal User'
+  }
+
+  if (googleId) {
+    googleId = 'Google User'
```
Comment (davidfudacz) [inline client/components/user/user-item.js]: same here

File: client/components/user/user-item.js [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+  } else {
+    googleId = 'Local User'
+  }
+
+console.log(props)
+  return (
+    <div className="productItem">
```
Comment (davidfudacz) [inline client/components/user/user-item.js]: userItem for class name

File: client/components/user/user-list.js [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+
+    componentDidMount() {
+        store.dispatch(getUsersFromServerThunkerator());
+    }
+
+    render() {
+        const users = this.props.allUsers;
+        // console.log(users);
+        return (
+            <div>
```
Comment (davidfudacz) [inline client/components/user/user-list.js]: class Name allUsers?  something like that

File: client/components/user/user-item.js (hunk 3 above)
Comment (dfeeney87) [reply client/components/user/user-item.js]: done

File: client/components/user/user-item.js (hunk 1 above)
Comment (dfeeney87) [reply client/components/user/user-item.js]: updated

File: client/components/user/user-item.js (hunk 2 above)
Comment (dfeeney87) [reply client/components/user/user-item.js]: done, updated to externalInternal

File: client/components/user/user-list.js (hunk 4 above)
Comment (dfeeney87) [reply client/components/user/user-list.js]: created, as discussed we should make generic list and item classes with user and product classes to compound

**Pull request merged**